src/main/resources/META-INF/hudson.remoting.ClassFilter

```diff
@@ -0,0 +1,9 @@
+# Model object classes, no deserialization logic
+com.amazonaws.regions.Region
```

Comment:
OK…was serializing these really intentional? Part of `config.xml` for a publisher or something I guess?

Reply:
yes, it's part of config.xml

Reply:
Yeah, it is a pretty sophisticated global publisher config. Decided not to touch that since the classes seem to be fine